packages/edit-site/src/components/site-hub/style.scss

```diff
@@ -33,6 +32,9 @@
 	text-decoration: none;
 	text-overflow: ellipsis;
 	white-space: nowrap;
+	display: flex;
+	flex-direction: row-reverse;
```

Comment:
The Button component supports a prop and value `iconPosition="right"` to swap the icon position. Please use that instead of custom CSS. Worth noting:
- The button component is already `inline-flex` and already uses `align-items: center;` so the CSS could be simplified. However...
- The removed `display: block;` is necessary and needs to be restored otherwise `text-overflow: ellipsis;` will not work. Screenshot to illustrate the missing ellipsis:

![Screenshot 2024-06-19 at 09 23 19](https://github.com/WordPress/gutenberg/assets/1682452/520aad70-f806-4b2c-bb57-959779816763)